Update PULL_REQUEST_TEMPLATE.md

philipp-bliznuk opened the pull request.

philipp-bliznuk committed:
Update PULL_REQUEST_TEMPLATE.md

iscaltritti commented:
Nice! I didn't realize we had access to this.

1. There's a redundant link at the end there
2. Optional description? Gotta give at least that!

I don't mind removing most of the template for SaaS, it makes sense to get it out of the way while we take care of consolidating its foundation and there's less people involver, but we will need it back eventually.

philipp-bliznuk committed:
Remove redundant link

philipp-bliznuk commented:
> Nice! I didn't realize we had access to this.
> 
> 1. There's a redundant link at the end there
> 2. Optional description? Gotta give at least that!
> 
> I don't mind removing most of the template for SaaS, it makes sense to get it out of the way while we take care of consolidating its foundation and there's less people involver, but we will need it back eventually.

Removed redundant link in the end, but for the rest I'm not sure what do you mean, I just removed the duplicates.

iscaltritti reviewed (approved):
Just that descriptions being optional rubs me the wrong way.

I was actually meaning to add more stuff to frontend, it'd be nice to get pictures or even videos of the new stuff, but again, since there are no releases here, that's fine.
